Applications.qmd

```diff
@@ -5,14 +5,44 @@ listing:
   id: Applications
   contents:
     - Applications/*.qmd
-  sort: "date desc"
   filter-ui: true
   page-size: 8
   fields: [title, description]
 ---
 
 
-This section provides an overview of the EO Applications available from Copernicus Data Space Ecosystem.
+Copernicus Data Space Ecosystem provides a wide range of applications that can be used to access, process, and visualize Copernicus data. These applications are designed for users of different skill level to be used for various applications.
+
+For new Copernicus dataspace ecosystem users we advise to explore the [Copernicus Data Space Browser](Applications/Browser.qmd) as starting point. In this platform users can explore various datasets, employing tools for visualization, comparison, and downloading with ease. 
+
+![](Applications/_images/SearchResult.png)
+
+From the browser it is possible to acces the [Data workspace](Applications/DataWorkspace.qmd), which is a tool to manage and and order satellite products. These products can then be further processed and downloaded for various purposes.
```

Comment:
I'm not sure if data workspace should be mentioned right after browser.
This looks like a very niche application, there's a list of processors which uses a bunch of acronyms that are hard to understand, and have no documentation.
Mention this app more towards the bottom.